eclass/autotools.eclass

```diff
@@ -4,7 +4,7 @@
 # @ECLASS: autotools.eclass
```

Comment:
I was told by its maintainers that there are some other changes for autotools.eclass pending, which should be done together with the EAPI 8 update. So I'd suggest that you drop it from the PR for now.

Reply:
I know. Time frame is about one month (or so I was told).